docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -28,11 +30,24 @@ Let's start by initializing a new Expo project, installing `@config-plugins/deto
     '$ cd eas-tests-example',
     '# Install @config-plugins/detox',
     '$ npm install --save-dev @config-plugins/detox',
-    '# Generate native code',
-    '$ npx expo prebuild',
   ]}
 />
 
+Now, open **app.json** and add the `@config-plugins/detox` plugin to your `plugins` list (this must be done before prebuilding). This'll automatically configure the Android native code to support Detox.
```

Comment:
```suggestion
Now, open **app.json** and add the `@config-plugins/detox` plugin to your `plugins` list (this must be done before prebuilding). This will automatically configure the Android native code to support Detox.
```